src/model/model.py

```diff
@@ -41,6 +41,14 @@ def load(self, path):
             data = nib.load(path).get_fdata()
         elif ext in ['.png', '.jpg']:
             data = imageio.imread(path)
+            if data.ndim == 3:
+                # remove alpha channel if same value everywhere
+                if data.shape[2] == 4 and (data[:, :, 3] == data[0, 0, 3]).all():
```

Comment:
I don't understand why you make the second condition in this case?

Reply:
I thought we should automatically remove the alpha channel only if alpha has the same value everywhere. Do you want to delete alpha channel in any case ?

Reply:
yes we do

Reply:
In general, image analysis does not really take account of the alpha channel, I think it's too sophisticated for what we want to do. We must not make it too complicated if it is not really used.

Reply:
Yep I agree, problem solved with last commit!